app/code/Magento/OfflineShipping/Model/ResourceModel/Carrier/Tablerate.php

```diff
@@ -220,7 +220,7 @@ private function deleteByCondition(array $condition)
      * @throws \Magento\Framework\Exception\LocalizedException
      * @return void
      */
-    private function importData(array $fields, array $values)
+    public function importData(array $fields, array $values)
```

Comment:
Such change may lead to data inconsistency due to lack of deleting old data:
https://github.com/magento/magento2/blob/121640f1cab195c0a1a44babf10bbaeebeeb7adb/app/code/Magento/OfflineShipping/Model/ResourceModel/Carrier/Tablerate.php#L277

Why need to make this method public as all what this method does is ```$this->getConnection()->insertArray($this->getMainTable(), $fields, $values);```?